src/status_im/contexts/wallet/sheets/network_selection/view.cljs

```diff
@@ -0,0 +1,76 @@
+(ns status-im.contexts.wallet.sheets.network-selection.view
+  (:require [quo.core :as quo]
+            [quo.foundations.resources :as quo.resources]
+            [react-native.core :as rn]
+            [status-im.contexts.wallet.common.utils :as utils]
+            [status-im.contexts.wallet.sheets.network-selection.style :as style]
+            [utils.i18n :as i18n]
+            [utils.money :as money]
+            [utils.re-frame :as rf]))
+
+(defn- network-item
+  [{:keys [network token on-select-network currency currency-symbol]}]
+  (let [{network-name :network-name
+         chain-id     :chain-id}                 network
+        {balances-per-chain :balances-per-chain} token
+        balance-for-chain                        (get balances-per-chain chain-id)
+        total-balance                            (money/bignumber (:balance balance-for-chain))
+        fiat-value                               (utils/calculate-token-fiat-value
+                                                  {:currency currency
+                                                   :balance  total-balance
+                                                   :token    token})
+        crypto-formatted                         (utils/get-standard-crypto-format token total-balance)
+        fiat-formatted                           (utils/get-standard-fiat-format crypto-formatted
+                                                                                 currency-symbol
+                                                                                 fiat-value)
+        token-symbol                             (:symbol token)
+        mainnet?                                 (= network-name :mainnet)]
+    [quo/network-list
+     {:label           (name network-name)
+      :network-image   (quo.resources/get-network network-name)
+      :token-value     (str crypto-formatted " " token-symbol)
+      :fiat-value      fiat-formatted
+      :on-press        #(on-select-network network)
+      :container-style (style/network-list-container mainnet?)}]))
+
+(defn view
+  [{:keys [token on-select-network]}]
+  (let [{token-networks :networks
+         token-symbol   :symbol} token
+        grouped-networks         (group-by #(= (:network-name %) :mainnet)
+                                           token-networks)
+        mainnet-network          (first (get grouped-networks true))
+        layer-2-networks         (get grouped-networks false)
```

Comment:
The best way to sort is by the `:layer` key in the network.

Reply:
@smohamedjavid I'm grouping using `:layer` key now. But right now I'm hardcoding `1` & `2`, do we have a constant for layers?

Reply:
No, @ajayesivan. Feel free to create one :+1: